src/app/classes/classes.component.html

```diff
@@ -1,15 +1,42 @@
-<h2>Class List</h2>
-<ul class="classes" style="list-style-type: none">
-  <li
-    *ngFor="let class of classes"
-    [class.selected]="class === selectedClass"
-    (click)="onSelect(class)"
-  >
-    {{ class.subject }} {{ class.course }}
-  </li>
-</ul>
-
-<div *ngIf="selectedClass">
-  <h2>Subscribed to {{ selectedClass.subject }} {{ selectedClass.course }}</h2>
-  <div>{{ selectedClass.id }}</div>
+<div class="container">
+  <div class="row">
+    <mat-form-field class="w-50">
+      <mat-label>Search Your Class</mat-label>
+      <input
+        matInput
+        type="text"
+        [(ngModel)]="searchText"
+        [ngModelOptions]="{ standalone: true }"
+        placeholder="ECE 49595"
+        (keyup)="classFilter()"
+      />
+    </mat-form-field>
+  </div>
+  <div *ngFor="let class of displayedClasses">
+    <div
+      *ngIf="
+        subscribedClases != undefined && !subscribedClases.includes(class.id)
+      "
+    >
+      <div class="row row-cols-3 py-2">
```

Comment:
Does this row-cols-3 even do anything?

Reply:
Yep, it keeps everything looking horizontal instead of vertical